openquake/hmtk/seismicity/declusterer/distance_time_windows.py

```diff
@@ -53,6 +53,7 @@
 from openquake.baselib.general import CallableDict
 
 TIME_DISTANCE_WINDOW_FUNCTIONS = CallableDict()
+DAYS = DAYS
 
```

Comment:
??

Reply:
yikes! this was the result of the "find and replace" of 364.75 with DAYS